src/shared/dumb_components/field.js

```diff
@@ -11,23 +11,23 @@ var Field = (props) =>
   ? <Input {...props} />
   : <div></div>
 
-var Display = ({ name, id, value, description }) =>
+var Display = ({ name, id, value, description, memberView }) =>
   <p>
     {description && name === 'News type: ' && <span><br /><i>{description}</i><br /></span>}
-    <br />
+    {memberView && <br />}
```

Comment:
Hm. This really doesn't belong in a 'display' component, but the id (for example) is already coupled to the member fields in a similar way. I will have a think about this